cSpell.json

```diff
@@ -402,9 +402,21 @@
             ]
         },
         {
-            "filename": "**/specification/migrate/resource-manager/Microsoft.Migrate/stable/2018-02-02/migrate.json",
+            "filename": "**/specification/migrate/resource-manager/**/*.json",
             "words": [
-                "Occured"
+                "Occured",
+                "Assessible",
+                "sqlcollectors",
+                "Reasonings",
+                "Overcommit",
+                "overcommit",
+                "SSDE",
+                "Recieved",
```

Comment:
>Recieved [](http://example.com/codeflow?start=17&length=8)

this is a misspelling, is it in a context where it can't be fixed, like an enum?

Reply:
looks like it ;(

Reply:
I don't see it in the old (Baseline) api though which suggests it is a new API - and we can fix the spelling?

Reply:
We have fixed any spelling mistake we could fix in our code. These ones are spelling mistakes in enums/ properties which have already been shipped. Now we cant change these properties as it would break the customers.

We have had API versions released between the last published one and this one, but we discussed with the ARM team/ breaking change team to release 1 clean API version and deprecate the old ones without releasing the swagger for those. Thats why you might not see these properties in old API versions, but sadly they have already been released a while ago

Reply:
Were those API versions preview, or GA versions? @kuchatur-ms

Reply:
There were both old GA and preview API versions.